infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -155,7 +160,8 @@ public SequencerServer(ServerContext serverContext) {
 
         globalLogTail = Address.getMinAddress();
         this.cache = new SequencerServerCache(config.getCacheSize(), globalLogTail - 1);
-        setUpTimerNameCache();
+        this.txResolutionTimer = MeterRegistryProvider.getInstance().map(registry ->
```

Comment:
Can you also capture histogram of the number of queues per transaction.

Reply:
Added